PSPs/drafts/psp-23.md

```diff
@@ -0,0 +1,517 @@
+# Wallet Interaction Standard
+
+- **PSP Number:** 23
```

Comment:
The psp number should be 29. Could you update this? Once this is done. I would be happy to merge this as an initial draft to hopefully also kickstart a discussion.